### ISSUE TITLE
Update document title based on view/route

### ISSUE THREAD

**Issue by ssciolla** [tl-its-umich-edu/my-learning-analytics#957]
Before single page apps, the user noticed that they changed pages by the title changes. I feel like updating the title would have benefit both to visual users as well as screen reader users as this is what's displayed in the tab. 

Apps like Gmail are a SPA and the title updates when you folders something from 

`Inbox - jonespm@gmail.com - Gmail`
`Starred - jonespm@gmail.com - Gmail`
or more specifically

`<Folder Name> <email> - Gmail`

It seems like our style should be of the format
`<Page Name> - <username> - My Learning Analytics`

Sorry for being a day late to the comments here. It's also possible the username is not needed.

_Originally posted by @jonespm in https://github.com/tl-its-umich-edu/my-learning-analytics/pull/949#issuecomment-639696065_

**Comment by chrisrrowland** [comment]
I have a solution for this using react-helmet, but this is only effective for standalone MyLA.  In LTI mode the title remains the name of the LTI tool.  This seems like it achieves the literal goal as stated by the issue, but I'm not sure that it achieves the spirit, as I'm not sure how it would be evaluated with regards to improving accessibility in LTI mode.  When I browse with a screen reader it doesn't read any title at all in LTI mode.

**Comment by jonespm** [comment]
Yeah, I don't think there's any way to fix this anymore with it running in an iframe in Canvas now over LTI.

**Comment by ssciolla** [comment]
Yeah, I hadn't thought about how this issue is changed in an LTI setting. Not sure we'd have much impact if we did implement the fix but it doesn't do anything with LTI. Hm...

**Comment by jonespm** [comment]
I think we might be doing the best we can now by having the title of the page appear in the <h1> tag, which it already was.

[It looks like](https://webaim.org/techniques/frames/#iframe) the "Title" attribute that Canvas passes on the <iframe> is what is read by most screen-readers. 

That title is just "Tool Content", which looks to be hardcoded in Canvas.

**Comment by chrisrrowland** [comment]
Is it worth adding the support for standalone or is it's time over

**Comment by chrisrrowland** [comment]
I added a PR just in case so I can move on.

**Comment by gsilver** [comment]
The <h1> is good enough. Thanks!

**Comment by gsilver** [comment]
Haha. The h1 is good enough! Thanks! @pushyamig
